cli/azd/pkg/project/framework_service_dotnet.go

```diff
@@ -82,6 +83,12 @@ func (dp *dotnetProject) Initialize(ctx context.Context) error {
 	return nil
 }
 
+func replaceUnderscoreWithColon(key string) string {
```

Comment:
This function name might not be extremely accurate, since the separator is a double underscore "__". I would just rename this as `normalizeDotNetSecret`